x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/selectors.ts

```diff
@@ -111,6 +111,11 @@ export const isOnPolicyDetailsPage = (state: Immutable<PolicyDetailsState>) => {
   );
 };
 
+/** Returns a boolean of whether the user is on the policy details page or not */
```

Comment:
this JSDoc comment does not look right for this selector.

Reply:
corrected